docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,341 @@
+be installed in your environment.
+
+```bash
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=rds
+export RELEASE_VERSION=$(curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REF="${SERVICE}-chart"
+export CHART_REPO="public.ecr.aws/aws-controllers-k8s/${CHART_REF}"
+export CHART_PACKAGE="${CHART_REF}-${RELEASE_VERSION}.tgz"
+
+mkdir -p "${CHART_EXPORT_PATH}"
+
+export ACK_SYSTEM_NAMESPACE=ack-system
+```
+
+Deploy the ACK service controller for Amazon RDS using the [Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart). You can download it to your
+workspace using the following commands:
+
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+helm pull "oci://${CHART_REPO}" --version "${RELEASE_VERSION}" -d "${CHART_EXPORT_PATH}"
+```
+
+Decompress and extract the Helm chart:
+
+```bash
+tar xvf "${CHART_EXPORT_PATH}/${CHART_PACKAGE}" -C "${CHART_EXPORT_PATH}"
+```
+
+You can now use the Helm chart to deploy the ACK service controller for
+Amazon RDS to your Amazon EKS cluster. At a minimum, you need to specify the
+AWS Region to execute the RDS API calls.
+
+```bash
+helm install --create-namespace \
+  --namespace "${ACK_SYSTEM_NAMESPACE}"  \
+  --set aws.region="$AWS_REGION" \
+  "ack-${SERVICE}-controller" "${CHART_EXPORT_PATH}/${SERVICE}-chart"
+
+helm list --namespace "${ACK_SYSTEM_NAMESPACE}" -o yaml    
+```
+
+### Configure IAM permissions
+
+Once the service controller is deployed, you will need to [configure the IAM permissions][irsa-permissions] for the controller to query the RDS API. For full details, please review the AWS Controllers for Kubernetes documentation for [how to configure the IAM permissions][irsa-permissions]. If you follow the examples in the documentation, use the value of `rds` for `SERVICE`.
+
+## Create an Aurora Serverless v2 PostgreSQL database
+
+To create an Aurora Serverless v2 database using the PostgreSQL engine, you must
+first create a DBSubnetGroup and a SecurityGroup for the VPC:
+
+```bash
+EKS_VPC_ID=$(aws eks describe-cluster --name "${EKS_CLUSTER_NAME}" --query "cluster.resourcesVpcConfig.vpcId" --output text)
+
+RDS_SUBNET_GROUP_NAME="mydbsubnetgroupack"
+RDS_SUBNET_GROUP_DESCRIPTION="mydb-subnetgroup"
+EKS_SUBNET_IDS=$(aws ec2 describe-subnets --filter "Name=vpc-id,Values=${EKS_VPC_ID}" --query 'Subnets[?MapPublicIpOnLaunch==`false`].SubnetId' --output text)
+
+cat <<-EOF > db-subnet-groups.yaml
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBSubnetGroup
+metadata:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ namespace: ${APP_NAMESPACE}
+spec:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ description: ${RDS_SUBNET_GROUP_DESCRIPTION}
+ subnetIDs:
+$(printf " - %s\n" ${EKS_SUBNET_IDS})
+ tags: []
+EOF
+
+kubectl apply -f db-subnet-groups.yaml
+
+RDS_SECURITY_GROUP_NAME="ACK_SG"
```

Comment:
nit: A bit weird that the name of the security group is all caps with underscores, whereas the other resources are snake case

Reply:
Modified.